pandas/core/computation/eval.py

```diff
@@ -5,17 +5,21 @@
 """
 
 import tokenize
+from typing import Any, Dict, Iterable, Union
 import warnings
 
+import numpy as np
+
 from pandas.util._validators import validate_bool_kwarg
 
+import pandas as pd
 from pandas.core.computation.engines import _engines
 from pandas.core.computation.scope import _ensure_scope
 
 from pandas.io.formats.printing import pprint_thing
 
 
-def _check_engine(engine):
+def _check_engine(engine: str) -> str:
```

Comment:
engine needs to be Optional[str] since `None` is valid here (cause of one CI failure)